src/crimson/osd/recovery_backend.cc

```diff
@@ -136,6 +136,25 @@ seastar::future<> RecoveryBackend::handle_backfill(
   }
 }
 
+seastar::future<> RecoveryBackend::handle_backfill_remove(
+  MOSDPGBackfillRemove& m)
+{
+  logger().debug("{} m.ls=", __func__, m.ls);
+  ceph_assert(m.get_type() == MSG_OSD_PG_BACKFILL_REMOVE);
```

Comment:
i'd suggest use `assert()` for smaller executable and less runtime overhead as i think this assert is mostly for document purpose.

Reply:
Yeah, that exactly was its purpose. `assert` is better for this kind of work. Thanks!